pkg/validation/types.go

```diff
@@ -36,3 +41,22 @@ type AffectedSchema struct {
 	Range   string `json:"range"`
 	FixedBy string `json:"fixedBy"`
 }
+
+// Time is a wrapper around time.Time.
+// The default UnmarshalJSON for time.Time expects the time.RFC3339 format,
+// which is not what is used in this repo.
+type Time struct {
+	time.Time
+}
+
+// UnmarshalJSON is inspired by the Go 1.18 (*time.Time).UnmarshalJSON implementation
+// https://cs.opensource.google/go/go/+/refs/tags/go1.18:src/time/time.go;l=1298.
+func (t *Time) UnmarshalJSON(data []byte) error {
+	if string(data) == "null" {
```

Comment:
`...|| string(data) == ""`

Reply:
I'm going to keep it as-is because this is what the official Golang implementation does